docs/refs/api/drivers/PwmOut.md

```diff
@@ -1,6 +1,12 @@
 #### PwmOut
 
-Use the PwmOut interface to control the frequency and mark-to-space ratio of a digital pulse train.
+Use the PwmOut interface to control the frequency and duty cycle of a PWM signal.
+
+**Tips:**
+
+* Set the cycle time first, and then set the duty cycle using either a relative time period via the write() function or an absolute time period using the pulsewidth() function. 
```

Comment:
Please rewrite this sentence for clarity. There are a lot of things using other things.